File: CodeChef/T_CT.cpp
#include<stdio.h>
#include<malloc.h>
#include<string.h>
int main(){
	int testCase,length;
	int alph[26],counter;
	char *c=(char *)malloc(sizeof(char)*100000);
	scanf("%d",&testCase);
	while(testCase--){
		for (int i = 0; i <26; alph[i++]=0);
		scanf("%s",c);
		length=strlen(c);
		counter=0;
		while (length--)
		{
			if(alph[c[length]-97]==0){
				alph[c[length]-97]++;
				counter++;
			}
		}
		if(counter%2==1)
			printf("Counter Terrorist\n");
		else
			printf("Terrorist\n");
	}
	
}

File: CodeChef/TCT.java
import java.util.Scanner;
class TCT{
	public static void main(String args[]){
		int testCase,counter=0,a[]=new int[26];
		Scanner in=new Scanner(System.in);
		testCase=in.nextInt();
		String s,ans="";
		char ch[];
		for (int i = 0; i < testCase; i++)
		{	a=new int[26];
			s=in.next();
			counter=0;
			ch=s.toCharArray();
			for (int j = 0; j <ch.length; j++)
			{	if(a[(int)ch[j]-97]==0){
					a[(int)ch[j]-97]++;
					counter++;
				}
			}
			if(counter%2==0){
				if(ans.equals(""))
					ans+="Terrorist";
				else
					ans+="\nTerrorist";
			}else{
				if(ans.equals(""))
					ans+="Counter Terrorist";
				else
					ans+="\nCounter Terrorist";
			}
			
			
		}
		System.out.println(ans);
	}
}
